test-network-nano-bash/orderer1.sh

```diff
@@ -17,8 +17,14 @@ export ORDERER_GENERAL_TLS_PRIVATEKEY="${PWD}"/crypto-config/ordererOrganization
 export ORDERER_GENERAL_TLS_CERTIFICATE="${PWD}"/crypto-config/ordererOrganizations/example.com/orderers/orderer.example.com/tls/server.crt
 # following setting is not really needed at runtime since channel config has ca root certs, but we need to override the default in orderer.yaml
 export ORDERER_GENERAL_TLS_ROOTCAS="${PWD}"/crypto-config/ordererOrganizations/example.com/orderers/orderer.example.com/tls/ca.crt
-export ORDERER_GENERAL_BOOTSTRAPMETHOD=file
-export ORDERER_GENERAL_BOOTSTRAPFILE="${PWD}"/channel-artifacts/genesis.block
+if [ $# -gt 0 ] && [ "$1" = "smartbft" ]
+then
+    export ORDERER_GENERAL_BOOTSTRAPMETHOD=none
+    export ORDERER_CHANNELPARTICIPATION_ENABLED=true
+else
+    export ORDERER_GENERAL_BOOTSTRAPMETHOD=file
```

Comment:
i think we are deprecating the file method in v3.0 no? so let's make it always none.